File: src/main/java/com/farm/service/NewsService.java
package com.farm.service;

import java.util.List;

import com.farm.model.News;
import com.farm.model.PageResult;

/**
 * service
 * 
 * @Date: 2019年05月14日 21:10:05 
 * @author 
 */
public interface NewsService extends BaseService<News> {
	
    public List<News> getByClassifyId(Object id);

    List<News> findAll();

    PageResult findPage(int page, int rows);

    News findOne(Integer id);

    void update(News news);

    void deleteByIds(Integer[] ids);

    PageResult findPage(News news, int page, int rows);

    List<News> getByTitle(String title);
}

File: src/main/webapp/js/service/newsService.js
//服务层
app.service('newsService',function($http){
	    	
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../content/findPage.do?page='+page+'&rows='+rows);
	}
	//删除
	this.dele=function(ids){
		return $http.get('../content/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../content/search.do?page='+page+"&rows="+rows, searchEntity);
	}
	//搜索
	this.getByClassifyId=function(treecode){
		return $http.post('../content/toFarmPage.do?treecode='+treecode);
	}
});
